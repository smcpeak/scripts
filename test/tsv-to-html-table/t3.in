name	address	pets
doug
jim	45 boardwalk avenue	123
