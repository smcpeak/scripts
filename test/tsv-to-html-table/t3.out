<table>
  <tr>
    <th>name
    <th>address
    <th>pets

  <tr>
    <td>doug
    <td>
    <td>

  <tr>
    <td>jim
    <td>45 boardwalk avenue
    <td class="numeric">123

</table>
